simtools/layout/array_layout.py

```diff
@@ -524,6 +546,63 @@ def export_telescope_list_table(self, crs_name):
 
         return table
 
+    def export_telescope_list_as_json(self, crs_name):
+        """
+        Return a single-telescope list as simtools-DB-style json.
+
+        Parameters
+        ----------
+        crs_name: str
+            Name of coordinate system to be used for export.
+
+        Returns
+        -------
+        dict
+            Dictionary with array element information.
+        """
+        table = self.export_telescope_list_table(crs_name)
+        if len(table) != 1:
+            raise ValueError("Only one telescope can be exported to json")
```

Comment:
If the function is supposed to work only with a single telescope, I would make that clearer in the name of the function. Or alternatively, I would expand the function to check the size of table and loop through the telescopes, exporting a json file to each one.